docs/blog/posts/2023-08-08-using_twarc.md

```diff
@@ -1,80 +1,82 @@
  conda environment with your collaborators, you can use `--prefix` argument instead of `-n` when calling `conda create`.
  You can also specify the version of python with which to make the environment.
  
-```
-$ conda create --prefix=/zfs/projects/<project-space>/conda/twitter python=3.10
+```title="Terminal Command"
+conda create --prefix=/zfs/projects/<project-space>/conda/twitter python=3.10
 ```
 
 In the above command, make sure you have the write privilages to `/zfs/projects/<project-space>/conda/` before making 
 the conda environment. After the conda env is created, all of the python packages including `twarc` will be installed there.
 
 After the environment is made, activate it:
 
-```
-$ source activate /zfs/projects/<project-space>/conda/twitter 
+```title="Terminal Command"
+source activate /zfs/projects/<project-space>/conda/twitter 
 ```
 
 Install the necessary python packages:
 
-```
-$ pip install pandas twarc python-dotenv
+```title="Terminal Command"
+pip install pandas twarc python-dotenv
 ```
 
-We will use [`python-dotenv`](https://pypi.org/project/python-dotenv/) package for passing our Twitter API key to our
+We will use [`python-dotenv`](https://pypi.org/project/python-dotenv/){:target="_blank"} package for passing our Twitter API key to our
 python script.  
 
 Last step is to make this conda environment into a Jupyter kernel.  
 
 Running the following command will install the active conda environment as a kernel in your JupyterHub. Pick a name
 for your kernel to go into the `--name` argument.
 
-```
-$ python -m ipykernel install --user --name=twitter
+```title="Terminal Command"
+python -m ipykernel install --user --name=twitter
 ```
 
-Start up [Jupyter](/yen/webBasedCompute.html) on any of the interactive yens (yen[1-5]) and you should now see a new
+Start up [Jupyter](/_getting_started/jupyter/) on any of the interactive yens (yen[1-5]) and you should now see a new
 kernel in the Launcher menu under Notebooks. Start a new notebook with that kernel.
 
-![](/images/twitter_kernel.png)
+![Twitter icon box highlighted in the Notebook section of JupyterHub](/assets/images/twitter_kernel.png)
 
 ## Scraping Twitter
 
 This guide uses Academic Research License discontinued by Twitter earler this year. 
 
-Once Academic Research application was approved, a researcher had access to [Twitter developer dashboard](https://developer.twitter.com/en/portal/products/academic-research).
+Once Academic Research application was approved, a researcher had access to [Twitter developer dashboard](https://developer.twitter.com/en/portal/products/academic-research){:target="_blank"}.
 Go to your Projects & App and make a new app. It will have a unique API Key and Secret and a Bearer Token that will be used to access Twitter data.
  
 ### Test Jupyter environment
 
 Make sure you can import all of the following python packages:
  
-```python
+```bash linenums="1" title="python"
```

Comment:
can we use .python as a language to get syntax highlights please? 

I think you can use:
```{.python .yaml linenums="1" title="python" }```